### PR TITLE
Update Templates - Scaffolded code

### PR DESCRIPTION
**Pull request by samruddhikhandale**

### CODE REVIEW

File: src/alpine/.devcontainer/devcontainer.json [hunk 1]
```diff
@@ -1,16 +1,19 @@
 {
 	"name": "Alpine",
-	"image": "mcr.microsoft.com/devcontainers/base:0-alpine-${templateOption:imageVariant}",
+	"image": "mcr.microsoft.com/devcontainers/base:0-alpine-${templateOption:imageVariant}"
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
```
Comment (samruddhikhandale) [inline src/alpine/.devcontainer/devcontainer.json:5]: Ordered properties by referencing https://github.com/devcontainers/spec/blob/main/schemas/devContainer.base.schema.json

File: src/alpine/.devcontainer/devcontainer.json [hunk 2]
```diff
@@ -1,16 +1,19 @@
 {
 	"name": "Alpine",
-	"image": "mcr.microsoft.com/devcontainers/base:alpine-${templateOption:imageVariant}",
+	"image": "mcr.microsoft.com/devcontainers/base:alpine-${templateOption:imageVariant}"
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
```
Comment (bamurtaugh) [inline src/alpine/.devcontainer/devcontainer.json:5]: Would it make sense to instead or additionally point to https://containers.dev/implementors/features/?

File: src/alpine/.devcontainer/devcontainer.json [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
 	// "postCreateCommand": "uname -a",
 
-	// Replace when using a ptrace-based debugger like C++, Go, and Rust
-	// "runArgs": [ "--init", "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
+	// Configure tool-specific properties.
+	// "customizations": {}
 
-	// Set `remoteUser` to `root` to connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
-	"remoteUser": "vscode"
+	// Comment out to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
```
Comment (bamurtaugh) [inline src/alpine/.devcontainer/devcontainer.json:17]: Should this be "Uncomment?"
```suggestion
	// Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
```

**Review by bamurtaugh — COMMENTED** [review]
Thanks!

File: src/alpine/.devcontainer/devcontainer.json (hunk 3 above)
Comment (samruddhikhandale) [reply src/alpine/.devcontainer/devcontainer.json]: Oops! changed.

**Review by chrmarti — APPROVED** [review]

**Review by joshspicer — APPROVED** [review]

**Pull request merged**